### ISSUE TITLE
phpunit fails on hhvm 3.6.1

### ISSUE THREAD

**Issue by mikehaertl** [travis-ci/travis-ci#3790]
This is probably related to the recent build environment update. I have a couple of projects that run tests with phpunit on HHVM. They all ran fine without problems until recently. I didn't change any configuration.

Here's an example: 
https://travis-ci.org/mikehaertl/php-tmpfile/jobs/60341036

The error is

```
$ phpunit
Fatal error: Class already declared: PHPUnit_Framework_Error in phar://phpunit-4.6.4.phar/phpunit/Util/../Framework/Error.php on line 22
The command "phpunit" exited with 255.
Done. Your build exited with 1.
```

**Comment by fpietka** [comment]
Same here

**Comment by spiritix** [comment]
Another example here:
https://travis-ci.org/spiritix/nested-set/jobs/60370441

**Comment by BanzaiMan** [comment]
Does HHVM 3.6.1 work with phpunit?

**Comment by mikehaertl** [comment]
@BanzaiMan Seems like a known issue: https://github.com/sebastianbergmann/phpunit/issues/1684

Someone there suggested this workaround: https://github.com/thephpleague/commonmark/commit/a0ef290ed4b4f95e7fa1a0bbc6af79c6ec6b5b1a

**Comment by BanzaiMan** [comment]
@mikehaertl Thanks for the research. I'm going to leave this open for now, although we have no immediate plans to update HHVM yet.

**Comment by mikehaertl** [comment]
> ... although we have no immediate plans to update HHVM yet.

Uhm, I don't quite understand. You *have* updated HHVM, otherwhise we would not have the current issues.

**Comment by BanzaiMan** [comment]
We [did](https://www.traviscistatus.com/incidents/yk24wrkb4082), and we [rolled back](https://www.traviscistatus.com/incidents/h91j8ft0qs2l). The current HHVM version is 3.5.0.

**Comment by BanzaiMan** [comment]
Hmm. I see 3.6.1 on standard infrastructure. I am looking into it. Sorry for the confusion.

**Comment by BanzaiMan** [comment]
OK. This should be fixed now.

**Comment by mikehaertl** [comment]
Looks good. My builds are green again. Thanks.

**Comment by photodude** [comment]
Seems the recent PHPUnit 4.6.6 is suppose to fix this issue. HHVM-3.6.2 LTS has also been released (unfortunately the news file for HHVM 3.6 lacks details on what changed).

**Comment by BanzaiMan** [comment]
I believe this can be closed now.
